internal/gatewayapi/testdata/clienttrafficpolicy-mtls-forward-client-cert.out.yaml

```diff
@@ -711,6 +716,7 @@ xdsIR:
     http:
     - address: 0.0.0.0
       headers:
+        generateRequestID: null
```

Comment:
why here is a `null`?

Reply:
The desire there was to leave the field omitted when left empty, I'll fix that now